app/controllers/status_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class StatusController < ApplicationController
+  def ping
+    json = {
+      'build_date' => ENV['APP_BUILD_DATE'] || 'Not Available',
+      'commit_id' => ENV['APP_GIT_COMMIT'] || 'Not Available',
+      'build_tag' => ENV['APP_BUILD_TAG'] || 'Not Available'
```

Comment:
Is it worth making the `'Not Available'` literals a constant here to avoid duplication?

Reply:
Yaas!